tests/e2e/go/e2e_test.go

```diff
@@ -0,0 +1,119 @@
+package main
```

Comment:
Again, its not clear why we need more than just Pytest for e2e. We can use https://github.com/testcontainers/testcontainers-go if we need integration tests from the existing test suites.

Reply:
Reverted.